spec/rspec/expectations/configuration_spec.rb

```diff
@@ -47,7 +45,13 @@ class << rspec_dup; undef configuration; end
       end
 
       context 'on an interpreter that does not provide BasicObject', :uses_should, :unless => defined?(::BasicObject) do
-        before { RSpec::Expectations::Syntax.disable_should(Delegator) }
+        around do |example|
+          in_sub_process_if_possible do
```

Comment:
In this case I think you can do away with the `around` hook entirely and just move it all into the example itself.